taqueria-flextesa-manager/lib.ts

```diff
@@ -8,6 +8,9 @@ import * as SanitizedArgs from './SanitizedArgs';
 
 import { Config, SandboxAccountConfig } from '@taqueria/protocol/taqueria-protocol-types';
 
+const protocolIdentifier = 'PtKathmankSp';
```

Comment:
What do you think of MACRO style naming for these global constants? It'd distinguish them from normal local constants which using camel case like what you have here. This makes it easy to know which is which when reading code that uses them.

Reply:
This is what I've adopted throughout my code at least.

Reply:
e.g. `PROTOCOL_HASH`

Reply:
Yes. Sure, Done